src/geometry/manifold/bump_function.lean

```diff
@@ -22,40 +22,9 @@ define a coercion to function for this type, and for `f : smooth_bump_function I
 The actual statements involve (pre)images under `ext_chart_at I f.c` and are given as lemmas in the
 `smooth_bump_function` namespace.
 
-We also define `smooth_bump_covering` of a set `s : set M` to be a collection of
-`smooth_bump_function`s such that their supports is a locally finite family of sets, and for each
-point `x ∈ s` there exists a bump function `f i` in the collection such that `f i =ᶠ[𝓝 x] 1`. This
-structure is the main building block in the construction of a smooth partition of unity (see TODO),
-and can be used instead of a partition of unity in some proofs.
-
-We say that `f : smooth_bump_covering I s` is *subordinate* to a map `U : M → set M` if for each
-index `i`, we have `closure (support (f i)) ⊆ U (f i).c`. This notion is a bit more general than
-being subordinate to an open covering of `M`, because we make no assumption about the way `U x`
-depends on `x`.
-
-We prove that on a smooth finitely dimensional real manifold with `σ`-compact Hausdorff topology,
-for any `U : M → set M` such that `∀ x ∈ s, U x ∈ 𝓝 x` there exists a `smooth_bump_covering I s`
-subordinate to `U`. Then we use this fact to prove a version of the Whitney embedding theorem: any
+ Then we use this fact to prove a version of the Whitney embedding theorem: any
 compact real manifold can be embedded into `ℝ^n` for large enough `n`.
```

Comment:
This is done in another file, right? If you want to keep this comment here, you should add a pointer to this other file.

Reply:
Removed.